### PR TITLE
Lock Terraform.exe version for tests and assert on bundled CLI version

### PR DESCRIPTION
**Pull request by flin-8**

### CODE REVIEW

File: source/Sashimi.Tests/BundledCliFixture.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+namespace Sashimi.Terraform.Tests
+{
+    [TestFixture]
+    public class BundledCliFixture
+    {
+        internal const string TerraformVersion = "0.11.8";
+
+        //Note that the CLI package may not end up in the test build folder when running locally.
+        [Test]
+        [WindowsTest]
```
Comment (flin-8) [inline source/Sashimi.Tests/BundledCliFixture.cs:21]: Only runs in [Windows](https://build.octopushq.com/project.html?projectId=OctopusDeploy_LIbraries_Sashimi_SashimiTerraform_CalamariNetFxWindowsTesting&testNameId=4237471779368549967&tab=testDetails)

**Comment by andrewabest** [comment]
Looks very neat 👌

**Review by andrewabest — APPROVED** [review]
This is a nice way to ensure if we want to bump this tool version, we need to change it in two places (i.e. really mean it). Great stuff 🥳

**Comment by andrewabest** [comment]
Just as I hit approve a couple of straggling build configs trickle in :sadpanda: 😆

**Comment by flin-8** [comment]
> Just as I hit approve a couple of straggling build configs trickle in :sadpanda: 😆

Yeah...the checks are funny..because they don't know a build is queued...

**Pull request merged**